src/main/java/roomescape/reservationtime/domain/ReservationTimeJdbcTemplateRepository.java

```diff
@@ -0,0 +1,72 @@
+package roomescape.reservationtime.domain;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.reservationtime.domain.entity.ReservationTime;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ReservationTimeJdbcTemplateRepository implements ReservationTimeRepository {
+    private final JdbcTemplate jdbcTemplate;
+
+    public ReservationTimeJdbcTemplateRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    private final RowMapper<ReservationTime> rowMapper = (resultSet, rowNum) -> {
+        return ReservationTime.of(
+                resultSet.getLong("id"),
+                resultSet.getString("start_at")
+        );
+    };
+    @Override
+    public List<ReservationTime> findAll() {
+        String sql = "select * from reservation_time";
+        return jdbcTemplate.query(sql, rowMapper);
+    }
+
+    @Override
+    public ReservationTime findById(Long id) {
+        String sql = "select * from reservation_time where id = ?";
+        return jdbcTemplate.queryForObject(sql, rowMapper, id);
+    }
+
+    @Override
+    public ReservationTime findByStartAt(String startAt) {
+        String sql = "select * from reservation_time where start_at = ?";
+        return jdbcTemplate.queryForObject(sql, rowMapper, startAt);
+    }
+
+    @Override
+    public Long countReservationMatchWith(Long id) {
+        String sql = "select count(*) "
+                + "from reservation as r "
+                + "inner join reservation_time as t "
+                + "on r.time_id = t.id "
+                + "where t.id = ?";
```

Comment:
```suggestion
        String sql = """
                    select count(*)
                    from reservation as r
                    inner join reservation_time as t on r.time_id = t.id
                    where t.id =?
                """;
```

전반적으로 sql string 에 [Java Multi-line String](https://www.baeldung.com/java-multiline-string) 을 적용하면 가독성을 개선할 수 있겠네요 :)